PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -209,11 +246,19 @@ for STRING in "$T1wFolder"/"$NativeFolder"@"$AtlasSpaceFolder"/"$NativeFolder"@n
   FolderI=`echo $STRING | cut -d "@" -f 1`
   FolderII=`echo $STRING | cut -d "@" -f 2`
   Mesh=`echo $STRING | cut -d "@" -f 3`
-  for STRINGII in MyelinMap_BC@dscalar SmoothedMyelinMap_BC@dscalar corrThickness@dscalar ; do
+
+  STRINGIIList="corrThickness@dscalar"
+  if [ "${T2wPresent}" = "YES" ] ; then
+    STRINGIIList+=" MyelinMap_BC@dscalar SmoothedMyelinMap_BC@dscalar"
+  fi
```

Comment:
Now here, we do indeed need to define a new list, so this is necessary.

Reply:
:-)